src/wallet/wallet.cpp

```diff
@@ -1907,7 +1907,7 @@ static void ApproximateBestSubset(vector<pair<CAmount, pair<const CWalletTx*,uns
     vfBest.assign(vValue.size(), true);
     nBest = nTotalLower;
 
-    seed_insecure_rand();
+    FastRandomContext rand;
```

Comment:
I'd prefer to keep the name at `FastRandomContext insecure_rand`, especially in the wallet code.